pylintrc

```diff
@@ -59,7 +59,9 @@ confidence=
 # --enable=similarities". If you want to run only the classes checker, but have
 # no Warning level messages displayed, use"--disable=all --enable=classes
 # --disable=W"
-#disable=
+
+# Temporary change to avoid E1136: Value 'Optional' is unsubscriptable
+disable=E1136
```

Comment:
I'd add this to `tox.ini` instead, and use a literal constant (I think it is `unsubscriptable-object`) instead of the error code. Also, `Union` triggers the same error, it's not just `Optional`.